sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_compiler.asciidoc

```diff
@@ -199,8 +201,20 @@ _Constraints:_ Available only when `PropertyListT` is an instance of
 other than those listed below in the section "New properties for the
 `create_kernel_bundle_from_source` function".
 
-_Effects:_ Creates a new kernel bundle that represents a kernel that is defined
-by the source code string `source` using the language `lang`.
+_Preconditions:_ There are two overloads of this function: one that reads the
+source code of the kernel from an `std::string`, and one that reads the source
```

Comment:
No changes required here, but an observation: "an `std::string`" reads better for people who say "an S-T-D string", while "a `std::string`" reads better for people who say "a stood string".  We should give some thought to what style we (and the SYCL specification) want to use.  Saying "an object of type `std::string`" would sidestep it.

Reply:
It appears that the consensus is to use "a std ..." (not "an std ..."), so I guess people have decided that the pronunciation is "stood" and not "s-t-d".

The C++ spec has 53 occurrences of "a std" and no occurrences of "an std".
The cppreference site has over 1300 occurrences of "a std" and only 71 occurrences of "an std".

I'll start using "a".

Reply:
These occurrences fixed in #12374.  Another occurrence fixed in this PR at 019977c420951beb16b8d927fbf917717979cb0a.